starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,130 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(min, max) {
+
+  if (min > max) {
+
+    return min;
+
+  } else {
+
+    return max;
+
+
+  }
+}
+
+console.log(`Iteration #1: The biggest number is: ${maxOfTwoNumbers(10, 15)}`);
+
+
+
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot', 'christmas'];
+
+
+function findLongestWord(wordArr) {
```

Comment:
Hey, Your code is pretty neat! One alternative to creating function is arrow function, in the future when you start to create functions inside Express or Classes, you will see how this will help you.
 I really like the tutorial from scrimba:  https://scrimba.com/p/p4Mrt9/c7vd3cd

Reply:
Thanks, i will look into it!